CHANGELOG.md

```diff
@@ -24,6 +24,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Incidents - Removed buttons column and replaced status with toggler ([#1237](https://github.com/grafana/oncall/issues/1237))
 - Responsiveness changes across multiple pages (Incidents, Integrations, Schedules) ([#1237](https://github.com/grafana/oncall/issues/1237))
+- Link to source was added ([1305](https://github.com/grafana/oncall-private/issues/1305))
```

Comment:
hmm, should we be adding links to `oncall-private`? This link is not publically accessible

Reply:
No, it should not, I will remove the link